packages/office-ui-fabric-react/src/components/Nav/examples/Nav.DisabledItems.Example.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Nav, INavLink } from 'office-ui-fabric-react/lib/Nav';
+import './Nav.Basic.Example.scss';
+
+export class NavDisabledItemsExample extends React.Component<any, any> {
+  public onLinkClick = (ev: React.MouseEvent<HTMLElement>, item?: INavLink) => {
+    if (item && item.name === 'News') {
+      alert('News link clicked');
+    }
+  };
+
+  public render(): JSX.Element {
+    return (
+      <div className="ms-NavExample-LeftPane">
+        <Nav
+          groups={[
+            {
+              links: [
+                {
+                  name: 'Home',
+                  url: 'http://example.com',
+                  disabled: true,
+                  links: [
+                    {
+                      name: 'Activity',
+                      url: 'http://msn.com',
+                      key: 'key1'
+                    },
+                    {
+                      name: 'MSN',
+                      url: 'http://msn.com',
+                      key: 'key2',
+                      disabled: true,
+                      altText: 'The link is temporary disabled'
+                    }
+                  ],
+                  isExpanded: true
+                },
+                { name: 'Documents', url: 'http://example.com', key: 'key3' },
+
+                {
+                  name: 'Unavailable Item',
+                  url: 'http://cnn.com',
+                  icon: 'News',
```

Comment:
same as in the vr test, can we make this `IconNames.News`?

Reply:
I see in IconNames enum the following comments
@deprecated Const enum use is deprecated. See github issue #7110: https://github.com/OfficeDev/office-ui-fabric-react/issues/7110

Should I still use it?

Reply:
Ah, good catch! I must've missed that. What you have is better then!